gnd/src/main/java/com/google/android/gnd/system/AuthenticationModule.java

```diff
@@ -0,0 +1,14 @@
+package com.google.android.gnd.system;
```

Comment:
Since there are several of these classes now, may be worth moving to their own package? (e.g., .gnd.system.auth?)

Reply:
Good plan. I'll do that.